src/App.vue

```diff
@@ -17,7 +16,7 @@
             <div class="mb-2">
               {{ currentUser }} Turn
             </div>
-            <div class="row horizontal-line ">
+            <div class="row horizontal-line background-blue">
               <div class="col-4 vertical-line">
                 <Tic :ticId="0" />
```

Comment:
used for loop if possible for tic component

Reply:
if i use for loop then my styling is break. so, for now i not use for loop